src/Microsoft.ML.ImageAnalytics/ImagePixelExtractorTransform.cs

```diff
@@ -542,28 +544,6 @@ private ValueGetter<VBuffer<TValue>> GetGetterCore<TValue>(IRow input, int iinfo
                         else
                         {
                             int idstMin = 0;
-                            if (ex.Alpha)
```

Comment:
>if (ex.Alpha) [](start = 28, length = 13)

why you delete this part?
is it useless?

Reply:
More or less. The pixel has an 'A' property, which already contains the value 0xFF if the image doesn't have alpha, so no need to special case.